packages/server/src/schema/uuid/abstract-uuid/resolvers.ts

```diff
@@ -378,10 +378,14 @@ async function resolveUuidFromDatabase(
               context,
             )
           : null
-      const subjectName = subject != null ? '/' + toSlug(subject.name) : ''
-      const slugTitle = baseUuid.entityTitle
-        ? toSlug(baseUuid.entityTitle)
-        : baseUuid.id
+      const subjectName =
+        subject != null && toSlug(subject.name).length > 0
+          ? toSlug(subject.name)
+          : 'serlo'
```

Comment:
```suggestion
      const subjectName = subject ? toSlug(subject.name) : 'serlo'
```
Would that not be enough?